_docs_integrate/25-requests-over-templates.md

```diff
@@ -0,0 +1,158 @@
+```
+
+You will receive a response with the complete RelationshipTemplate. In this guide we will only use the Templates truncated reference and its id. Thus make sure to save the `truncatedReference` and the `id` for the next steps.
+
+```jsonc
+{
+    "id": "RLT..",
+    // ...
+    "truncatedReference": "UkxU..."
+}
+```
+
+## Load the Template on the Requestor Connector
+
+Now you have to load the Template on the Requestor Connector. You can do so by calling the `POST /api/v2/RelationshipTemplates/Peer` route with the following content. Use the truncated reference you copied before:
+
+```jsonc
+{
+    "reference": "UkxU..."
+}
+```
+
+If no Relationship exists this will trigger a process in the Request module. The Request module will create a new incoming Request on that we will work in the next step. You can observe this by long polling the incoming Requests or by waiting for the `consumption.incomingRequestReceived` event.
+
+The long polling is done by calling the `GET /api/v2/Requests/Incoming` route. You can use the query params `source.reference=<id-of-the-template>` and `status=ManualDecisionRequired` to filter only for Requests that belong to the Template you are currently working on.
+
+For more information about the events you can head over to the [Connector Modules site]({% link _docs_integrate/03-connector-modules.md %}) and read about the `AMQP Publisher` and the `WebhooksV2` module that are handling events.
+
+If you got the Request save the `id` of the Request for the next step.
+
+## Answer the Request
+
+The rejection is explained before the acceptance because you can re-do it as often if you want. If you accept the Request a RelationshipRequest will be sent and no new Request will be created until the RelationshipRequest is answered. If it is accepted the Request module will recognize the existing Relationship and will also not create a new Request.
+
+We therefore recommend to first reject the Request and then accept it if you want to test the full flow.
+
+If there is no open RelationshipRequest or existing Relationship, you can trigger the creation of a new Request by [loading the Template again](#load-the-template-on-the-requestor-connector) with the same truncated reference.
+
+### Reject
+
+If you want to reject the Request you can do so by calling the `POST /api/v2/Requests/Incoming/{id}/Reject` route. You can use the `id` you saved in the previous step. In the payload you have to reject all RequestItems. In case of the example Request the payload is the following:
+
+```jsonc
+{
+    "items": [
+        {
+            "accept": false
+        }
+    ]
+}
+```
+
+You receive a Request in status `Decided`. This is where the Request module steps in and handles the Request based on you decision. Because you rejected the Request, the Request module will only move the Request to status completed. This behavior can be observed by querying the Request again after a short waiting time (`GET /api/v2/Requests/Incoming/{id}`).
```

Comment:
> You receive a Request in status `Decided`

Ich glaube, das ist verwirrend. Man empfängt den Request ja eigentlich nicht. Vielleicht "As a result, the Request moves to status `Decided`"

Reply:
Das ist falsch. Du bekommst als payload ein request im status decided.